plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/common/core/utils/X509CertificateParser.java

```diff
@@ -74,10 +76,10 @@ private String getIssuer(X509Certificate certificate) {
 		}
 
 		private String getValidity(X509Certificate certificate) {
-			SimpleDateFormat dateFormat = new SimpleDateFormat("E, d MMM yyyy HH:mm:ss");
+			SimpleDateFormat dateFormat = new SimpleDateFormat(DATE_FORMAT);
 			return new StringBuilder()
 				.append("Issued On: ").append(dateFormat.format(certificate.getNotBefore())).append('\n')
```

Comment:
dare to move "Issued On: " to a constant for the sake of consistency (with EXPIRES_ON_PREFIX)?